frontend/src/data/api-service.ts

```diff
@@ -120,100 +170,66 @@ export const createApiService = ({
     return config
   })
   client.interceptors.response.use(
-    (response) => response,
+    (response) => {
+      logger.debug(
+        ">>>",
```

Comment:
What is this `>>>` used in the debug logs in this module? Is that a Consola thing?

Reply:
I used it so that the API request is easy to see in the logs. Do you think it's not necessary, or there's a better way to display this?
I have often had problems when debugging the server rendering: has the request been fired, have we received a response, or was there some other problem during the API request/response phase? It's easy to debug the browser-based requests, but for there usually is no information about the server requests, so I decided to add it here.

Reply:
No, that's great, I'm moreso asking just about the `>>>` specifically. I suppose it's just there to add a prefix to the logs? I haven't tried the PR to see whether the logs have an implicit format (timestamp prefix, etc). I'm just a little wary of adding haphazard logging, even debug logs. I'd be thrilled to have logs of outbound requests from SSR in production even, but the `>>>` isn't a parseable prefix :sweat_smile: 

We have this issue to use structured logging via Consola: https://github.com/WordPress/openverse/issues/3435 we can address the issue then. It would be better not to add things like `>>>` to begin with (if indeed it isn't a special signifier for Consola) but it's not an important issue. We can resolve that in the issue.

Reply:
Removed `>>>`.
We can replace the `src`'s logger with a structured logging Consola setup in a follow-up PR.